kokoro/ubuntu/sign.sh

```diff
@@ -8,9 +8,38 @@ set -x
 echo ${KOKORO_GFILE_DIR}
 cd $KOKORO_GFILE_DIR
 mkdir -p signed && chmod 777 signed
-/escalated_sign/escalated_sign.py -j /escalated_sign_jobs -t signjar \
- $KOKORO_GFILE_DIR/artifacts.jar \
- $KOKORO_GFILE_DIR/signed/artifacts.jar
-/escalated_sign/escalated_sign.py -j /escalated_sign_jobs -t signjar \
- $KOKORO_GFILE_DIR/content.jar \
- $KOKORO_GFILE_DIR/signed/content.jar
\ No newline at end of file
+mkdir -p signed/plugins && chmod 777 signed/plugins
+mkdir -p signed/features && chmod 777 signed/features
+ 
+cp $KOKORO_GFILE_DIR/artifacts.jar $KOKORO_GFILE_DIR/signed/artifacts.jar
+cp $KOKORO_GFILE_DIR/content.jar $KOKORO_GFILE_DIR/signed/content.jar
+cp $KOKORO_GFILE_DIR/category.xml $KOKORO_GFILE_DIR/signed/category.xml
```

Comment:
Where is content.xml?

But you definitely don't need category.xml? We've had it there since I think GPE. If it's superfluous I'll drop it.